library/kani_macros/src/contract.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Utility functions for macro expansion of function contract clauses.
+use proc_macro2::TokenStream as TokenStream2;
```

Comment:
Why `2`?

Reply:
Because `TokenStream` exists in both `proc_macro` and `proc_macro2` crate. I've redefined the type to `TokenStream2` when it comes from `proc_macro2`. Most other crates that use `proc_macro2` do something similar - they refer to the types from `proc_macro2` using the suffix `2`.